### PR TITLE
Code lint with Flake8

### PR DESCRIPTION
**Pull request by skystar-p**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,12 +8,13 @@ matrix:
...
       install:
         - pip install django coverage coveralls
-        - pip install -r $(pwd)/backend/requirements.txt
+        - pip install -r $(pwd)/backend/requirements-dev.txt
       before_script:
         - export PROJECT_PATH=$(pwd)/backend/backend
         - python3 util/secret-key-gen.py
       script:
         - cd backend
+        - flake8 --config ./flake8
```
Comment (tirr-c) [inline .travis.yml]: :+1:

**Comment by skystar-p** [comment]
squashed one commit

**Comment by tirr-c** [comment]
@web-mighty-homu r+

**Comment by web-mighty-homu** [comment]
:pushpin: Commit b131da7 has been approved by `tirr-c`

**Comment by web-mighty-homu** [comment]
:hourglass: Testing commit b131da795ec14892409a42ea984042b5e0792976 with merge e3d31c7391020bdb15862dae6df17813b972753a...

**Comment by web-mighty-homu** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/swsnu/swpp17-team2/builds/299464668?utm_source=github_status&utm_medium=notification)
Approved by: tirr-c
Pushing e3d31c7391020bdb15862dae6df17813b972753a to master...

**Pull request merged**